src/QsFmt/Formatter/SyntaxTree/Node.fsi

```diff
@@ -37,6 +37,9 @@ module internal Trivia =
     /// </summary>
     val newLine: Trivia
```

Comment:
It might be good to update the documentation for `newLine` to clarify that it's a new line node containing the default new line character for the current platform.